### PR TITLE
Atoms: Fix link css in Explainer atoms 

### PR DESCRIPTION
**Pull request by ajwl**
## What does this change?

This does three things: 

1) Removes the `/temporary/_snippets.scss` file  that is not needed any more & the two places it was imported.

2) Replaces the unused code from `/temporary/_explainers.scss` with a small amount of CSS that fixes a link styling problem. 

3) Imports `_explainers.scss` into `head.content.garnett.scss` so it is pulled in on article pages. 

Link styling problem visible in explainer here: https://www.theguardian.com/us-news/2017/feb/16/deutsche-bank-examined-trump-account-for-russia-links

## Screenshots

Problem: 
screen shot 2018-07-20 at 18 37 00

Fixed:
screen shot 2018-07-20 at 18 17 49

Fixed hover state: 
screen shot 2018-07-20 at 18 17 58
 

### Tested

- [x ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Comment by ajwl** [comment]
[Bug: Explainer atom links override pillar styles](https://trello.com/c/EHIEMiU1/343-bug-explainer-atom-links-override-pillar-styles)

File: static/src/stylesheets/module/atoms/temporary/_explainers.scss [hunk 1]
```diff
@@ -1,158 +1,9 @@
-// This will will be removed when the atom library is handling
-// the rendering of all snippet atoms
+//style support for legacy explainer atoms
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/atoms/temporary/_explainers.scss]: Can you move that file out of the `temporary` please?

Comment (ajwl) [reply static/src/stylesheets/module/atoms/temporary/_explainers.scss]: Okay, can do

**Review by regiskuckaertz — APPROVED** [review]
Great 👍 Normally these fixes would go into the atom library, but since this is a flaw due to the CSS in frontend, it is fine add it here.

**Comment by philmcmahon** [comment]
Code removal and a fix, nice!! 🎉

**Pull request merged**